nilearn/conftest.py

```diff
@@ -484,6 +484,33 @@ def surf_label_img(surf_mesh):
     return SurfaceImage(surf_mesh(), data)
 
 
+@pytest.fixture
+def surf_maps_img(surf_mesh):
```

Comment:
maybe put this one in nilearn/maskers/tests/test_surface_maps_masker.py

unless we use the fixture in more than one subpackage it may be easier to keep the fixture 'scope' as small as possible

This may be something to clean more globally as part of the surface fixture refactoring.